judge/models/problem.py

```diff
@@ -94,36 +94,52 @@ def add_problem_i18n_name(self, key, language, name_field=None):
     description = models.TextField(verbose_name=_('problem body'))
-    authors = models.ManyToManyField(Profile, verbose_name=_('creators'), blank=True, related_name='authored_problems')
+    authors = models.ManyToManyField(Profile, verbose_name=_('creators'), blank=True, related_name='authored_problems',
+                                      help_text=_('These users will be able to edit the problem, '
+                                                  'and be listed as authors.'))
     curators = models.ManyToManyField(Profile, verbose_name=_('curators'), blank=True, related_name='curated_problems',
-                                      help_text=_('These users will be able to edit a problem, '
+                                      help_text=_('These users will be able to edit the problem, '
                                                   'but not be publicly shown as an author.'))
     testers = models.ManyToManyField(Profile, verbose_name=_('testers'), blank=True, related_name='tested_problems',
                                      help_text=_(
-                                         'These users will be able to view a private problem, but not edit it.'))
-    types = models.ManyToManyField(ProblemType, verbose_name=_('problem types'))
-    group = models.ForeignKey(ProblemGroup, verbose_name=_('problem group'), on_delete=CASCADE)
+                                         'These users will be able to view the private problem, but not edit it.'))
+    types = models.ManyToManyField(ProblemType, verbose_name=_('problem types'),
+                                    help_text=_('The type of problem '
+                                                '(e.g. Data Structures, Simple Math, etc.).'))
+    group = models.ForeignKey(ProblemGroup, verbose_name=_('problem group'), on_delete=CASCADE,
+                                help_text=_('The group of your problem, '
+                                            'shown as Category in the problem list '
+                                            '(e.g. Fun Math, Seasonal, etc.). '
+                                            'Often used if your problem is from a contest.'))
     time_limit = models.FloatField(verbose_name=_('time limit'),
                                    help_text=_('The time limit for this problem, in seconds. '
                                                'Fractional seconds (e.g. 1.5) are supported.'))
     memory_limit = models.IntegerField(verbose_name=_('memory limit'),
                                        help_text=_('The memory limit for this problem, in kilobytes '
                                                    '(e.g. 64mb = 65536 kilobytes).'))
     short_circuit = models.BooleanField(default=False)
-    points = models.FloatField(verbose_name=_('points'))
+    points = models.FloatField(verbose_name=_('points'),
+                                help_text=_('Points given for problem completion. '
```

Comment:
* Using `""` for the outer quotes so you don't need to escape `'` is generally considered better style.
* I'd word that last part as "Points are displayed with a 'p' suffix if partial."
* "given" sounds a bit odd. Maybe "awarded"?

Reply:
Fixed